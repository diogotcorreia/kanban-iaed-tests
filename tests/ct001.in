t 1 Description
t 1 Description
l 9
n -3
n 0
u a
u a
m 3 a DONE
m 1 b DONE
m 1 a WRONG
m 1 a DONE
m 1 a TO DO
m 1 a DONE AND DONE
d IN ACTIVITY
a TO DO
a wRONG ACTIVITY
a WRONG ACTIvITY
a WRONG ACTIVITy
a 4
a 5
a 6
a 7
a 8
a 9
a 10
a 11
q
