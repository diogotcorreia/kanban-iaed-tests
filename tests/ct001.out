task 1
duplicate description
9: no such task
invalid time
0
user already exists
no such task
no such user
no such activity
duration=0 slack=-1
task already started
no such activity
no such activity
duplicate activity
invalid description
invalid description
invalid description
too many activities
